nightwatch.conf.js

```diff
@@ -0,0 +1,62 @@
+const seleniumServer = require('selenium-server');
+const phantomjs = require('phantomjs-prebuilt');
+const chromedriver = require('chromedriver');
+const geckodriver = require('geckodriver');
+
+require('nightwatch-cucumber')({
+    cucumberArgs: [
+        '--require', 'test/e2e/step_definitions',
+        '--format', 'pretty',
+        '--format', 'json:test/e2e/reports/cucumber.json',
+        '--format-options', '{"colorsEnabled":true}',
+        'test/e2e/features'
+    ]
+});
+
+module.exports = {
+    output_folder: 'test/e2e/reports',
+    live_output: true,
+    test_workers: {
+        enabled: true,
+        workers: 'auto'
```

Comment:
This is not needed right now, because there is only one test. 

> "Test concurrency is done at the file level. Each test file will fill a test worker slot. Individual tests/steps in a test file will not run concurrently." - http://nightwatchjs.org/guide#parallel-running